AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/EViewHolder.java

```diff
@@ -155,6 +156,6 @@ public JFieldVar getAlreadyInflated() {
 	}
 
 	private void setAlreadyInflated() {
-		alreadyInflated = generatedClass.field(PRIVATE, JType.parse(codeModel(), "boolean"), "alreadyInflated_", JExpr.FALSE);
+		alreadyInflated = generatedClass.field(PRIVATE, JType.parse(codeModel(), "boolean"), "alreadyInflated" + generationSuffix(), JExpr.FALSE);
```

Comment:
You should also used this in the JavaDoc which is added because of this field. You can leave that constant, and replace `_` with `generationSuffix()`.